homeassistant/components/vicare/sensor.py

```diff
@@ -132,6 +133,134 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
+        native_unit_of_measurement=VOLUME_CUBIC_METERS,
+        value_getter=lambda api: api.getGasSummaryConsumptionHeatingCurrentMonth(),
+        unit_getter=lambda api: api.getGasSummaryConsumptionHeatingUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="gas_summary_consumption_heating_currentyear",
+        name="Heating gas consumption current year",
+        native_unit_of_measurement=VOLUME_CUBIC_METERS,
+        value_getter=lambda api: api.getGasSummaryConsumptionHeatingCurrentYear(),
+        unit_getter=lambda api: api.getGasSummaryConsumptionHeatingUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="gas_summary_consumption_heating_lastsevendays",
+        name="Heating gas consumption last seven days",
+        native_unit_of_measurement=VOLUME_CUBIC_METERS,
+        value_getter=lambda api: api.getGasSummaryConsumptionHeatingLastSevenDays(),
+        unit_getter=lambda api: api.getGasSummaryConsumptionHeatingUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="hotwater_gas_summary_consumption_heating_currentday",
+        name="Hot water gas consumption current day",
+        native_unit_of_measurement=VOLUME_CUBIC_METERS,
+        value_getter=lambda api: api.getGasSummaryConsumptionDomesticHotWaterCurrentDay(),
+        unit_getter=lambda api: api.getGasSummaryConsumptionDomesticHotWaterUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="hotwater_gas_summary_consumption_heating_currentmonth",
+        name="Hot water gas consumption current month",
+        native_unit_of_measurement=VOLUME_CUBIC_METERS,
+        value_getter=lambda api: api.getGasSummaryConsumptionDomesticHotWaterCurrentMonth(),
+        unit_getter=lambda api: api.getGasSummaryConsumptionDomesticHotWaterUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="hotwater_gas_summary_consumption_heating_currentyear",
+        name="Hot water gas consumption current year",
+        native_unit_of_measurement=VOLUME_CUBIC_METERS,
+        value_getter=lambda api: api.getGasSummaryConsumptionDomesticHotWaterCurrentYear(),
+        unit_getter=lambda api: api.getGasSummaryConsumptionDomesticHotWaterUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    ViCareSensorEntityDescription(
+        key="hotwater_gas_summary_consumption_heating_lastsevendays",
+        name="Hot water gas consumption last seven days",
+        native_unit_of_measurement=VOLUME_CUBIC_METERS,
+        value_getter=lambda api: api.getGasSummaryConsumptionDomesticHotWaterLastSevenDays(),
+        unit_getter=lambda api: api.getGasSummaryConsumptionDomesticHotWaterUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
```

Comment:
Isn't this sensor a moving window? As in, every day, the last 7 days would shift, right?

Reply:
I cannot answer that since the sensor is not provided by my device.
@Styx85 or @killer0071234 can you answer that?

Does this sensor cover the last 7 days or is it more like the current week?

Reply:
@oischinger 
Hi for my Device "Vitovalor PT 2" it is the current week (mo-su). I just did a Screenshot in the app to reassure how it is reported.
![Screenshot_2022-04-26-20-40-40-996_com viessmann vicare](https://user-images.githubusercontent.com/32103696/165370004-6acb4a4e-9969-47f2-ad29-e20c23fa4584.jpg)

Reply:
@oischinger 
As I mentioned here #https://github.com/somm15/PyViCare/issues/246 the Viessmann app seems to use three different api calls / reportings and does the rest as internal calculation. If it helps to clarify what is reported.

Reply:
Can you show a screenhot of the "days"-section?
does it show the last seven days or monday-Today?

Reply:
It shows the last seven days and additionaly the current day.
![Screenshot_2022-04-26-21-14-55-433_com viessmann vicare](https://user-images.githubusercontent.com/32103696/165375511-d7412e8b-4125-4bca-bfd8-1ffa19a96b9c.jpg)

Reply:
I wonder if this sensor is actually useful in context of home assistant where you would rather use e.g. the consumption/year.
Either we remove it or we remove the `SensorStateClass.TOTAL_INCREASING`

Reply:
IMHO, you need only 1 total_increasing for the energy dashboard. It doesn't matter much which one.

Reply:
I wonder if this sensor is actually useful in context of home assistant where you would rather use e.g. the consumption/year.
Either we remove it or we remove the `SensorStateClass.TOTAL_INCREASING`

Reply:
Correct, some like to reproduce the app display in ha as well.
Anyhow. With these changes the Vitovalor still is missing the total gas consumption. Or am i missing something?

Reply:
I removed the "last seven days" sensor since it's the most problematic one concerning `SensorStateClass.TOTAL_INCREASING`

The others might be useful to someone